extensions/azurecore/src/azureResource/tree/flatAccountTreeNode.ts

```diff
@@ -220,6 +221,10 @@ class FlatAccountTreeNodeLoader {
 					}
 				}
 			}
+			// Create "No Resources Found" message node if no resources found under azure account.
+			if (this._nodes.length === 0) {
+				this._nodes.push(AzureResourceMessageTreeNode.create(localize('azure.resource.flatAccountTreeNode.noResourcesLabel', "No Resources found"), this._accountNode))
```

Comment:
Lower case the 'R' in resources.

Reply:
I have it capitalized to be consistent with the "No Subscriptions found" message here: https://github.com/microsoft/azuredatastudio/issues/22373#issuecomment-1476581202 but I guess that's something we can discuss whether we want these capitalized or not.

Reply:
In that case, consistency is better and let's keep it.